app/src/main/java/org/fossasia/phimpme/gallery/activities/SecurityActivity.java

```diff
@@ -131,8 +131,8 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
                     title.setText("Choose folders to secure");
                     RecyclerView recyclerView = (RecyclerView) view.findViewById(R.id.secure_folder_recyclerview);
                     recyclerView.setLayoutManager(new LinearLayoutManager(getApplicationContext()));
-                    SecureDialogAdapter s = new SecureDialogAdapter();
-                    recyclerView.setAdapter(s);
+                    SecureDialogAdapter securedLocalFolders = new SecureDialogAdapter();
```

Comment:
If possible make this final.

Reply:
I think there is no need to make it final as the adapter is not changing after the initialization.

Reply:
It speeds up execution if you do so.
https://developer.android.com/training/articles/perf-tips.html

Reply:
Yes, you are right, declaring final does improve performance. But in local variable which is declared and accessed only once, that difference in performance will be negligible in this case.

Reply:
1 maybe negligible compared to 1000. But 1+1+1+1+1+1... (100 times may not). It was just a suggestion though. I am okay to merge this at all costs :)

Reply:
Well it was a nice suggestion even I learned something.

Reply:
Ping @codedsun